plugin/core/types.py

```diff
@@ -187,6 +187,7 @@ class Settings:
     diagnostics_additional_delay_auto_complete_ms = None  # type: int
     diagnostics_delay_ms = None  # type: int
     diagnostics_gutter_marker = None  # type: str
+    diagnostics_highlight_style = None  # type: Union[str, Dict[str, str]]
```

Comment:
Should this be `""` instead of `None`? Should fix the mypy lint complaints.

Reply:
It must remain `None` because from what I understand, this is the way to annotate class member variables in old versions of python.

Reply:
see https://mypy.readthedocs.io/en/stable/class_basics.html

Reply:
Ah I see. Mypy is complaining later (line 280) when I set the var to user input directly, and leave the verification for `_style_str_to_flag`. Should I add a directive for mypy to ignore that line?